### ISSUE TITLE
Index Out of Bounds Exception [crash]

### ISSUE THREAD

**Issue by aneidos** [avetharun/OriginalFur#7]
Sometimes the game crashes when respawning. It happens occasionally with this mod active, after clicking on the "Respawn" button:
```
Description: Rendering entity in world

java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
	at com.studiopulsar.feintha.originalfur.fabric.client.FurRenderFeature.render(FurRenderFeature.java:103)
	at com.studiopulsar.feintha.originalfur.fabric.client.FurRenderFeature.method_4199(FurRenderFeature.java:65)
        ...
```
The issue seems to be caused by this line of code, added in the last version, in the `class FurRenderFeature` under `public void render(...)`:
`o = iPEM.originalFur$currentOrigins()[0];`
https://github.com/avetharun/OriginalFur/blob/21fc7d52910dde49dda5af8b7990e3a2213702d8/src/main/java/com/studiopulsar/feintha/originalfur/fabric/client/FurRenderFeature.java#L103C12-L103C54

The game crashes when `originalFur$currentOrigins()` returns an empty list, so an easy fix would be:
```
var co = iPEM.originalFur$currentOrigins();
if (co.isEmpty()) {return;}
o = co[0];
```

**Comment by avetharun** [comment]
Should be fixed next update

**Comment by avetharun** [comment]
Fixed as of [1.0.7](https://modrinth.com/mod/origin-furs/version/IEYNCqyS)

Sorry for the delay

**Comment by avetharun** [comment]
Note: original 1.0.7 update had a crash, so I need to remake it. Sorry!
